core/src/main/java/tech/underoaks/coldcase/loader/enums/TileContents.java

```diff
@@ -14,7 +14,8 @@ public enum TileContents {
     PLAYER(3, Player.class),
     MOVABLE_BLOCK(4, movableBlock.class),
     TEST_ITEM(5,TestItem.class),
-    TEST_ITEM02(6,TestItem02.class);
+    TEST_ITEM02(6,TestItem02.class),
+    GOALOBJECT(7,GoalObject.class);
```

Comment:
```suggestion
    GOALOBJECT(8,GoalObject.class);
```

kommtz sonst zu merge konflikten mit dev